skan/test/test_csr.py

```diff
@@ -125,6 +126,18 @@ def test_pixel_values():
     assert_almost_equal(stats.loc[0, 'mean pixel value'], expected)
 
 
+def test_maximum_length():
+    image = np.random.random((45,))
+    expected = np.max(image[1:-1])
+    stats = csr.summarise(image)
+    print (csr.path_maximum_length())
+    print (stats)
+    #print (stats.loc[0, 'max-length'], expected)
+    #assert_almost_equal(stats.loc[0, 'max-length'], expected)
```

Comment:
As above: you need to make a slightly more interesting test case for this, just by varying the values inside the skeleton.